### DISCUSSION TITLE
Adding conversion tracking code to receipt pages

### DISCUSSION THREAD

**Discussion by derekashauer** [impress-org/givewp#7206] [Developer Q&A]
I am using a brand new install of GiveWP, so the latest version using the new form builder.

I am trying to add some simple JS to the receipt page to add some conversion tracking to it. I cannot seem to find any way to do this or hook into the DonatonReceipt.tsx template file.

Any ideas?

Thanks!

**Comment by JoshuaHungDinh** [comment]
Hi @ashwebstudio !

Welcome to GiveWP! We're thrilled to have you on board.

As a new user I would like to let you know, we offer a convenient [Google Analytics add-on](https://givewp.com/documentation/add-ons/google-analytics/). This add-on provides integration for tracking your GiveWP donations by form.

If you have a specific tool in mind, we'd love to assist further. Could you share a bit more about the particular conversion tracking you would like to use or any additional details on your implementation that might help us assist you better?

**Comment by derekashauer** [comment]
I am building https://conversionbridgewp.com which will integrate many other analytic platforms beside just GA. I am trying to find a way to make that happen but not yet having any success. Ideally I need to be able to add some options to the donation form settings to enable conversion tracking and set some custom tracking labels as well.

Any way to get a copy of that add-on so I can see how it was done there and replicate it?

**Comment by kjohnson** [comment]
Hey @ashwebstudio - I've been following your work on X/Twitter.

I sent you a read permission invite to the [Google Analytics](https://github.com/impress-org/give-google-analytics) add-on repository for reference.

See `src/Donations` for v2 compatibility and `src/FormExtension` for v3 compatibility with the Visual Donation Form Builder.

Feel free to ping me here if you have any questions.

**Comment by derekashauer** [comment]
This is great, thanks! I will take a look and let you know if I have any questions.
